app/decorators/io_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class IoDecorator < Hydra::Derivatives::IoDecorator
+  def original_filename
```

Comment:
No - it seems more like the difference between a rails uploaded file and a file on disk. Hydra::Works/CurationConcerns expects an uploaded file.